92-tuned.install

```diff
@@ -19,20 +19,15 @@ LOADER_ENTRIES="$BOOT_ROOT/loader/entries"
 
 [ "$COMMAND" = "add" ] || exit 0
 
-# Workaround for rhbz#1657858
+# The zipl bootloader doesn't support variables
 ARCH=`uname -m`
-[ "${ARCH:0:4}" = "s390" ]
-HANDLE_INITRD="$?"
+[ "${ARCH:0:4}" = "s390" ] && exit 0
 
 pushd "$LOADER_ENTRIES" &> /dev/null
 for f in `basename "$MACHINE_ID"`-*.conf; do
   if [ -f "$f" -a "${f: -12}" != "-rescue.conf" ]; then
     grep -q '^\s*options\s\+.*\$tuned_params' "$f" || sed -i '/^\s*options\s\+/ s/\(.*\)/\1 \$tuned_params/' "$f"
-    if [ "$HANDLE_INITRD" = "1" ]; then
-      grep -q '^\s*initrd\s\+.*\$tuned_initrd' "$f" || sed -i '/^\s*initrd\s\+/ s/\(.*\)/\1 \$tuned_initrd/' "$f"
-    else
-      sed -i '/^\s*initrd\s\+.*\$tuned_initrd/ s/\s\+\$tuned_initrd\b//g' "$f"
-    fi
+    sed -i '/^\s*initrd\s\+.*\$tuned_initrd/ s/\s\+\$tuned_initrd\b//g' "$f"
```

Comment:
I guess this line should be:
```
grep -q '^\s*initrd\s\+.*\$tuned_initrd' "$f" || sed -i '/^\s*initrd\s\+/ s/\(.*\)/\1 \$tuned_initrd/' "$f"
```
Otherwise it would break initrd handling on non zipl machines.

Reply:
> I guess this line should be:

I think you are right. I've updated the commit now. I wasn't able to get a s390x machine so couldn't test this version.